.github/workflows/cd.yml

```diff
@@ -63,6 +63,18 @@ jobs:
 
       - name: Print Environment Information
         run: npx nx report
+      
+      - name: Format affected packages
+        run: npx nx affected --target=format
+
+      - name: Lint affected packages
+        run: npx nx affected --target=lint
+
+      - name: Test affected packages
+        run: npx nx affected -t test
+
+      - name: Build affected packages
+        run: npx nx affected -t build
```

Comment:
Can we merge these commands with the `run-many` step on line 80?
It looks like we're duplicating lint,test,build here so I think we can add just the `format` step to the list on line 80 and then we can delete these steps.

Reply:
Good point @nickhealGDS - amended and pushed!